core/translations/i18n/en.json

```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```

Comment:
This is actually deliberate. Although the project is now called Concierge, the modules are still Kassy modules for convenience. (Its harder to spell `concierge.json` than `kassy.json` and I can't be bothered renaming every single one).